### DISCUSSION TITLE
Alphabetize menu items on Win 11

### DISCUSSION THREAD

**Discussion by orbops** [Open-Shell/Open-Shell-Menu#997] [Q&A]
When I first installed Open Shell, all the programs were alphabetized. After a couple of months, it seems like all new programs are being listed alphabetically below the 'Apps' folder. Is there a way I can alphabetize the 'All Programs' listings?

**Comment by ghost** [comment]
Did you figure this out?

**Comment by tallman69** [comment]
I have not found a setting to realphabetize the list, so I drag and drop them where I want them in the list.

**Reply by orbops** [reply]
That's what I had to do also.

**Comment by ghost** [comment]
I found the setting. You have to right click one of the programs and click
“Auto Arrange” in the context menu.

**Comment by tallman69** [comment]
@TheGloriousDuck 

Great!  Thanks!  I played around with the two options "Sort by Name" and "Auto Arrange".  If you click on "Sort by Name" it will alphabetize the list once, then you can move items manually if you like.  If you select "Auto Arrange" it will alphabetize the list, and keep it alphabetized when new items are added, and it is not possible to rearrange the list manually.
